.gitconsensus.yaml

```diff
@@ -0,0 +1,46 @@
+# See https://github.com/gitconsensus/GitConsensusCLI for more information about this file.
+
+version: 3
+
+# Don't count any vote from a user who votes for multiple options
+prevent_doubles: true
+
+# Add extra labels for the vote counts and age when merging
+extra_labels: true
+
+
+pull_requests:
+
+  # How many votes (including abstains) that are required.
+  quorum: 10
+
+  # Required percentage of "yes" votes (ignoring abstentions). It's a good idea to give "no" votes more power.
+  threshold: 0.74
+
+  # Number of hours after last action (commit or opening the pull request) before issue can be merged
+  # 15 days.
+  merge_delay: 375
+
+  # Number of votes at which the merge_delay gets ignored, assuming no negative votes.
```

Comment:
This comment talks about number of votes, but the next line is a delay at 10 days?